scripts/js/lib/fs.ts

```diff
@@ -14,6 +14,8 @@ import fs from "fs/promises";
 
 import { $ } from "zx/core";
 
+import { readFile } from "fs/promises";
```

Comment:
We already import fs on line 13. You should remove this line. On line 40, use `fs.readFile`